Sources/NetworkService/NetworkError/NetworkError.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+/// Contains network errors that may occur, while the application is running.
+public enum NetworkError: Error, CaseIterable {
+    /// Indicates that the Internet connection has been lost.
+    case lostConnection
```

Comment:
Could you please clarify when the `lostConnection` is triggered and when the `requestTimeOut` is triggered? And how we're going to distinguish these trigger events?

Reply:
I cannot answer this question because I do not know how we would distinguish between these two cases. 
This case has been removed.